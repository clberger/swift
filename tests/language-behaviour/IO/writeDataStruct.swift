type file;

type S { string l; string c; string r; }

S s;

file f <"writeDataStruct.out">;

f=writeData(s);

s.l = "baz";
s.c = "BAZ";
s.r = "Baz";

