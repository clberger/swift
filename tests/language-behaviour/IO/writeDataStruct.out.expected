r c l 
Baz BAZ baz 
